config/core.json

```diff
@@ -2,8 +2,6 @@
 	"features": {
 		"activity-panels": true,
 		"analytics": true,
-		"analytics-dashboard": true,
```

Comment:
Same question here- would the `analytics-dashboard` be included in `analytics` or could these ever be considered separate toggleable features?